pyproject.toml

```
@@ -24,14 +24,13 @@ dependencies = [
   "PyGithub>=1.58.1",
   "sphinx>=5.3.0",
   "breathe>=4.34.0",
-  "sphinx-design>=0.3.0",
+  "myst-parser>=1.0.0",
```

```suggestion
  "myst-parser[linkify]>=1.0.0",
```
You dropped the "linkify" feature, this is causing the CI to fail.

---

Thanks. Fixed in [fc71a63](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/140/commits/fc71a634547fdfdf89c9a0c81fc64c4671b45628)